fp.js

```diff
@@ -1,28 +1,34 @@
 // Convert the following code to use `.map`
 
-var peoplesAges = []
+// var peoplesAges = people.map(function(p){
+//     var age = [];
+//     age = p.age;
+//     return age;
+//   })
```

Comment:
this would work! Basically `.map` will return a new array. In order to use `.map` you need to pass a callback as an argument, then the parameter of the callback is a variable representing *each* item in the array. i.e.

```js
var peoplesAges = people.map(function(person) {
 return person.age
})
```